.cirrus.star

```diff
@@ -30,6 +30,6 @@ def main(ctx):
         return []
 
     if "[cd build]" in commit_msg or "[cd build cirrus]" in commit_msg:
-        return fs.read(arm_wheel_yaml)
+        return fs.read(arm_wheel_yaml) + fs.read(arm_tests_yaml)
```

Comment:
If I understand correctly, we previously only build the wheel and install it but did not test it. Regarding the testing, we only test one of the python versions?

Reply:
Previously tagging with `[cd build]` would only trigger the build the wheels **and** tests on all supported Python versions. The `arm_tests_yaml` is a Linux ARM test environment similar to the Azure CI jobs and nothing to do with wheel building. With this PR, `arm_tests_yaml` would also trigger with the `[cd build]` tag. 

Concretely, looking at the [Cirrus job](https://cirrus-ci.com/build/6402402991996928) for this PR, the jobs tagged with `CIBW_BUILD` are the wheel builds and `linux_aarch64_test` is the Linux ARM test.